public/main.js

```diff
@@ -0,0 +1,87 @@
+const { app, BrowserWindow } = require('electron');
+const isDev = require('electron-is-dev');
+const path = require('path');
+const os = require('os');
+const bodyParser = require('body-parser');
+const expressApp = require('express')();
+const server = require('http').Server(expressApp);
+const io = require('socket.io')(server);
+
+if (isDev) {
+  console.log('Running in development');
+} else {
+  console.log('Running in production');
+}
+
+let mainWindow;
+
+function createWindow() {
+  expressApp.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
Express now has it's own "bodyParser" so we can replace bodyParser completely with express.urlencoded({extended: true}) and express.json() (will have to change how express is required for this to work i.e. const express = require('express'); const expressApp = express();)

Reply:
agreed.